src/docs/product/discover-queries/index.mdx

```diff
@@ -82,11 +82,21 @@ Share your queries as often as you want. You can share URLs with other users who
 
 ### Delete Queries
 
-On the **Discover** page, each saved query card has an ellipsis that will open a context menu. From here, you can delete the query. This action is irreversible. You can also delete the query within the Query Results view by clicking the trash can in the upper right.
+On the **Discover** page, each saved query card has an ellipsis that will open a context menu. From here, you can delete the query. This action is irreversible. You can also delete the query within the [query results](#query-results) view by clicking the "Delete Saved Query" option in the context menu.
 
 ### Add to Dashboard
 
-Queries can also be added to <SandboxLink scenario="dashboards" projectSlug="react">custom dashboards</SandboxLink> as widgets. You can find the "Add to Dashboard" button by opening the context menu in the **Discover** page or within the [query results](#query-results) view. When you click "Add to Dashboard", a form opens where you can name your widget, select your target dashboard, and make other changes before saving your new widget.
+Queries can also be added to <SandboxLink scenario="dashboards" projectSlug="react">custom dashboards</SandboxLink> as widgets. You can find the "Add to Dashboard" button by opening the context menu in the **Discover** page or within the context menu of the [query results](#query-results) view. When you click "Add to Dashboard", a form opens where you can name your widget, select your target dashboard, and make other changes before saving your new widget.
+
+### Set as Default
+
+<Include name="early-adopter-note.mdx" />
+
+If your organization is part of the Early Adopter program, when you navigate to **Discover**, the homepage opens the query builder where you can immediately begin creating a new query. By default, the page displays the pre-built "All Events" query. You can customize your homepage by configuring how your columns and charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded.
```

Comment:
One last nit: we're not just configuring how the chart appears, we're possibly changing the query, right? Made the following suggestion.

```suggestion
If your organization is part of the Early Adopter program, when you navigate to **Discover**, the homepage opens the query builder where you can immediately begin creating a new query. By default, the page displays the pre-built "All Events" query. 

You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded.
```

Reply:
Yep! Everything you can set in a discover query, you can now save it for the homepage (except for the title because each case is a new query). I think this new copy makes sense and I've updated it